modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/OptionFieldKeyValue/OptionFieldKeyValue.es.js

```diff
@@ -0,0 +1,251 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayIcon from '@clayui/icon';
+import ClayPanel from '@clayui/panel';
+import {ClayTooltipProvider} from '@clayui/tooltip';
+import classNames from 'classnames';
+import {normalizeFieldName} from 'data-engine-js-components-web';
+import {sub} from 'frontend-js-web';
+import React, {useRef, useState} from 'react';
+
+import {FieldBase} from '../FieldBase/ReactFieldBase.es';
+import Text from '../Text/Text.es';
+import {useSyncValue} from '../hooks/useSyncValue.es';
+
+import './OptionFieldKeyValue.scss';
+
+const Main = ({
```

Comment:
nit: instead of using a function component and adding displayName below. Just change this to 
`export default function OptionFieldKeyValue() {`